tests/programs/custom/cv32e40x_csr_access_test/cv32e40x_csr_access_test.S

```diff
@@ -401,491 +401,5931 @@ _start0:
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0x5a5a5a5a
-	csrrw x14, 784, x7
+	li x7, 0x14a5a5a5a
+	csrrw x14, 2949, x7
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0xdcd7df1c
-	csrrw x14, 784, x7
+	li x7, 0xf858ba3a
+	csrrw x14, 2949, x7
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
 	li x7, 0xa5a5a5a5
-	csrrs x14, 784, x7
+	csrrs x14, 2949, x7
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0x5a5a5a5a
-	csrrs x14, 784, x7
+	li x7, 0x14a5a5a5a
+	csrrs x14, 2949, x7
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0x4284010d
-	csrrs x14, 784, x7
+	li x7, 0xb90115cb
+	csrrs x14, 2949, x7
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
 	li x7, 0xa5a5a5a5
-	csrrc x14, 784, x7
+	csrrc x14, 2949, x7
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0x5a5a5a5a
-	csrrc x14, 784, x7
+	li x7, 0x14a5a5a5a
+	csrrc x14, 2949, x7
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0x686e59ae
-	csrrc x14, 784, x7
+	li x7, 0x65032be1
+	csrrc x14, 2949, x7
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrwi x14, 784, 0b00101
+	csrrwi x14, 2949, 0b00101
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrwi x14, 784, 0b11010
+	csrrwi x14, 2949, 0b11010
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrwi x14, 784, 0b00000
+	csrrwi x14, 2949, 0b11111
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrsi x14, 784, 0b00101
+	csrrsi x14, 2949, 0b00101
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrsi x14, 784, 0b11010
+	csrrsi x14, 2949, 0b11010
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrsi x14, 784, 0b11001
+	csrrsi x14, 2949, 0b01000
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrci x14, 784, 0b00101
+	csrrci x14, 2949, 0b00101
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrci x14, 784, 0b11010
+	csrrci x14, 2949, 0b11010
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	csrrci x14, 784, 0b10001
+	csrrci x14, 2949, 0b00100
 	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	# mcountinhibit
+	# mhpmcounter6
 	li x7, 0xa5a5a5a5
-	csrrw x14, 800, x7
-	li x7, 0x0000000d
+	csrrw x14, 2822, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0x5a5a5a5a
-	csrrw x14, 800, x7
-	li x7, 0x00000005
+	li x7, 0x14a5a5a5a
+	csrrw x14, 2822, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0x21e1dd8a
-	csrrw x14, 800, x7
-	li x7, 0x00000008
+	li x7, 0x1351c1f3
+	csrrw x14, 2822, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
 	li x7, 0xa5a5a5a5
-	csrrs x14, 800, x7
-	li x7, 0x00000008
+	csrrs x14, 2822, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0x5a5a5a5a
-	csrrs x14, 800, x7
-	li x7, 0x0000000d
+	li x7, 0x14a5a5a5a
+	csrrs x14, 2822, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0xd384db72
-	csrrs x14, 800, x7
-	li x7, 0x0000000d
+	li x7, 0xc4d56e65
+	csrrs x14, 2822, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
 	li x7, 0xa5a5a5a5
-	csrrc x14, 800, x7
-	li x7, 0x0000000d
+	csrrc x14, 2822, x7
+	li x7, 0x00000000
 	bne x7, x14, csr_fail
-	li x7, 0x5a5a5a5a
-	csrrc x14, 800, x7
-	li x7, 0x00000008
```

Comment:
I dont understand this change. It seems like you change some value to 0, it that the purpose?